.idea/modules.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/PureOS.iml" filepath="$PROJECT_DIR$/.idea/PureOS.iml" />
```

Comment:
This is just annoying fix for broken CLion config (broken during product split months ago)